mite/__main__.py

```diff
@@ -48,6 +49,7 @@
     --recorder-socket=SOCKET          Socket [default: tcp://127.0.0.1:14306]
     --delay-start-seconds=DELAY       Delay start allowing others to connect [default: 0]
     --volume=VOLUME                   Volume to run journey at [default: 1]
+    --duration=SECONDS                Duration to run journey for [default: 60]
```

Comment:
IIUC, this changes the interface of `mite journey test` to stop after 60 seconds by default...I'm skeptical about making this change to the interface.

Reply:
I can delete if we want, but struggled to see if a journey would ever stop as we had it before, if it doesn't have an exhaustive data pool (does it?). I figured this was probably better than a never-ending test for all of these journeys. But it's likely another lack of understanding thing.

Reply:
My use case for this is often to leave it running indefinitely while I look at logs or something in another console, or wait for an error to hit so I can drop into the debugger, or something.  I just hit ctrl-c when I want it to stop.

Reply:
Removed